src/System.Net.NameResolution/tests/PalTests/NameResolutionPalTests.cs

```diff
@@ -32,6 +32,17 @@ public void GetHostByName_LocalHost()
             Assert.NotNull(hostEntry.Aliases);
         }
 
+        public static object[][] InvalidHostNames = new object[][] {
+            new object[] { "fkdsfsjfsrH:" },
+            new object[] { "..." }
+        };
+
+        [Theory, MemberData(nameof(InvalidHostNames))]
+        public void GetHostByName_Invalid_HostName(string hostName)
```

Comment:
We usually have a 3-part naming for our test:

<MethodOrPropertyBeingTested>_<Conditions>_<Expectations>

So, this test would be better named as: "GetHostByName_InvalidHostName_Throws"